README.md

```diff
@@ -46,72 +46,44 @@ You can then keep `betterdiscordctl` up to date with one command:
 
   Increases the verbosity level, for progressively more debugging information.
 
-* `-s` / `--scan` (default `/opt:/usr/share:/usr/lib64`)
-
-  Changes the directories scanned for Discord installations. These are scanned
-  in the order provided. Note that these do **not** end in `/discord`—if your
-  Discord installation is at `/opt/discord`, then `/opt` should be scanned.
-
 * `-f` / `--flavors` (default `:canary:ptb`)
 
   When scanning, looks for installations with the given suffixes (case
   insensitive, both hyphenated and unhyphenated). Stable is `''`, as it has no
   suffix. Note that **no** spaces follow colons. Your Discord flavor probably
   doesn't have a space in it, so don't use any in here.
 
-* `-d` / `--discord` (requires `--modules`)
-
-  Skip scanning and use the Discord installation directory specified. This
-  **does** probably end in `/discord`.
-
 * `-m` / `--modules`
 
   Disregards scanning results and uses the specified modules directory (found
   inside Discord's user-specific storage directory).
 
 * `-r` / `--bd-repo` (default `https://github.com/rauenzi/BetterDiscordApp`)
 
-  When installing BetterDiscord, use the specified Git repository. Does _not_
-  affect updates. Defaults to Zerebos's BandagedBD fork.
+  When installing BetterDiscord, use the specified Git repository.
 
-* `--bd-repo-branch` (default `injector`)
+* `--bd-repo-release` (default `latest`)
 
-  When downloading from `--bd-repo`, use this branch.
+  When downloading from `--bd-repo`, use this release.
 
 * `-b` / `--betterdiscord`
 
-  Instead of maintaining a local clone of BetterDiscord, use the specified
-  directory.
-
-* `-c` / `--copy-bd`
-
-  Instead of using a symbolic link, copy the BetterDiscord directory into
-  Discord's modules.
+  Instead of downloading a copy of `betterdiscord.asar`, use the specified file.
```

Comment:
"Instead of downloading from a release, use the specified `betterdiscord.asar` file."